client/extensions/woocommerce/state/ui/products/variations/edits-reducer.js

```diff
@@ -22,7 +31,7 @@ function editProductVariationAction( edits, action ) {
 	const _edits = prevEdits.map( ( productEdits ) => {
 		if ( productId === productEdits.productId ) {
 			found = true;
-			const variationId = variation && variation.id || nextBucketIndex( productEdits[ bucket ] );
+			const variationId = variation && variation.id || { index: Number( uniqueId() ) };
```

Comment:
Number isn't supported in iE11 - will that be a problem? http://caniuse.com/#feat=es6-number